wordpress-mobile/WordPress-iOS
# Fix `TicketFieldIDs` on 32-bit devices.

Fixes #n/a

To test:
- Run on a 32-bit device (ex: iPhone 5). Verify there are no errors caused by `TicketFieldIDs`.
- Run on a 64-bit device. Verify there are no errors caused by `TicketFieldIDs`.
